node/tessel-export.js

```diff
@@ -249,30 +249,40 @@ Tessel.Port = function(name, socketPath, board) {
 
   this.pwm = [];
 
-  this.I2C = function I2C(address, mode) {
-    return new Tessel.I2C({
+  this.I2C = function I2CInit(address, mode) {
+    if (port._i2c) {
+      port._i2c.disable();
+    }
+
+    port._i2c =  new Tessel.I2C({
```

Comment:
So, I pulled this branch and ran the tests, which showed me that I had already written tests that support the semantics that I described. These tests all fail with this new code, and they absolutely should not fail: 


```
Running "nodeunit:tests" (nodeunit) task
Testing disable-port-initialization.js........OK
Testing tessel.js....................F..............FF.............................F.....

// ... irrelevant test failures elided

>> Tessel.Port.prototype - multiplePortsI2C
>> Error: Expected 11 assertions, 0 ran
>> at Object.exports.Tessel.Port.prototype.tearDown (test/unit/tessel.js:453:5)

>> Tessel.Port.prototype - multiplePortsI2C
>> TypeError: port._i2c.disable is not a function
>> at new I2CInit (tessel-export.js:254:17)
>> at Object.exports.Tessel.Port.prototype.multiplePortsI2C (test/unit/tessel.js:580:20)
>> at Object.exports.Tessel.Port.prototype.setUp (test/unit/tessel.js:447:5)

>> Tessel.I2C - enableOnceOnly
>> Error: undefined == false
>> at Object.exports.Tessel.I2C.enableOnceOnly (test/unit/tessel.js:1322:10)
>> at Object.exports.Tessel.I2C.setUp (test/unit/tessel.js:1310:5)

>> Tessel.I2C - enableOnceOnly
>> Error: undefined == true
>> at Object.exports.Tessel.I2C.enableOnceOnly (test/unit/tessel.js:1336:10)
>> at Object.exports.Tessel.I2C.setUp (test/unit/tessel.js:1310:5)

>> Tessel.I2C - enableOnceOnly
>> Error: 2 == 1
>> at Object.exports.Tessel.I2C.enableOnceOnly (test/unit/tessel.js:1337:10)
>> at Object.exports.Tessel.I2C.setUp (test/unit/tessel.js:1310:5)
```